### PR TITLE
Added new events for Phone Usage Events

### PR DESCRIPTION
**Pull request by this-Aditya**

### CODE REVIEW

File: commons/passive/phone/phone_user_interaction.avsc [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "name": "PhoneInteractionState",
       "type": "enum",
       "doc": "Interaction type with an Android device.",
-      "symbols": ["STANDBY", "UNLOCKED", "SHUTDOWN", "BOOTED", "OTHER"]
-      }, "doc": "What interaction state the phone has." }
+      "symbols": ["STANDBY", "UNLOCKED", "SHUTDOWN", "BOOTED", "OTHER", "UNKNOWN"]
+      }, "doc": "What interaction state the phone has.", "default": "UNKNOWN" }
```
Comment (yatharthranjan) [inline commons/passive/phone/phone_user_interaction.avsc:14]: is this default required?

**Review by yatharthranjan — APPROVED** [review]
LGTM

File: commons/passive/phone/phone_user_interaction.avsc (hunk 1 above)
Comment (this-Aditya) [reply commons/passive/phone/phone_user_interaction.avsc:14]: "Shouldn't I 😅? Shall I undo these changes?"

Comment (mpgxvii) [reply commons/passive/phone/phone_user_interaction.avsc:14]: It depends if they are provided by Android?

Comment (this-Aditya) [reply commons/passive/phone/phone_user_interaction.avsc]: Fine, Updated It.

**Review by mpgxvii — APPROVED** [review]
LGTM

**Pull request merged**